app/assets/stylesheets/tai.scss

```diff
@@ -202,3 +202,26 @@ hr {
 .banner-links {
     color: #005ea5;
 }
+
+#headingInLabel {
+    display: table;
+    table-layout: fixed;
+    width: 100%;
+
+    .form-label{
+        font-family: "nta",Arial,sans-serif;
```

Comment:
How come we specify the font name in tai? Is it not defined globally? Plus arial is purely microsoft as far as I know.

Reply:
yeah you are right we dont need that. i had copied h1 styles as they were. I have removed it